src/main/kotlin/LocalEvaluationClient.kt

```diff
@@ -84,6 +84,10 @@ class LocalEvaluationClient internal constructor(
         }
     }
 
+    fun stop() {
+        deploymentRunner.stop()
```

Comment:
I think we need to make clear to customer that this stop is a shutdown, that it can't be started again. In `DeploymentRunner.stop`, it calls `FlagUpdater.shutdown`, which, for flag poller, it halts executor service and cannot schedule polling again. 
If starting later is needed, then `DeploymentRunner.stop` should call `flagUpdater.stop`. 
Also, `DeploymentRunner.stop` might need to acquire the same lock as the `DeploymentRunner.start`.